src/ui/components/orientation_arrows.c

```diff
@@ -17,6 +17,13 @@
 #include <hardfault.h>
 #include <util.h>
 
+#if !defined(TESTING)
+#include <qtouch.h>
+#else
+void qtouch_force_calibrate(void);
```

Comment:
this should move here:
https://github.com/digitalbitbox/bitbox02-firmware/blob/pr%2F4/test/unit-test/framework/mock_qtouch.c#L22

Reply:
And then include mock_qtouch, like here:

https://github.com/digitalbitbox/bitbox02-firmware/blob/f8c348a9ab61175b06a348d6b7d17c3c41c8cc84/src/touch/gestures.c#L24